app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate params[:session][:password]
```

Comment:
`user&.authenticate params[:session][:password]`
You can study more about ruby safe ravigation

Reply:
Thanks, I get it